examples/notebooks/beam-ml/bigtable_enrichment_transform.ipynb

```diff
@@ -603,6 +603,37 @@
         "\n"
       ]
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### What is a Cross-Join?\n",
+        "A cross-join is a Cartesian product operation where each row from one table is combined with every row from another table. It is useful when we want to create all possible combinations of two datasets.\n",
+        "\n",
+        "**Example:**\n",
+        "- Table A:\n",
+        "  | A1 | A2 |\n",
+        "  |----|----|\n",
+        "  |  1 |  X |\n",
+        "  |  2 |  Y |\n",
+        "\n",
+        "- Table B:\n",
+        "  | B1 | B2 |\n",
+        "  |----|----|\n",
+        "  | 10 |  P |\n",
+        "  | 20 |  Q |\n",
+        "\n",
+        "**Result of Cross-Join:**\n",
+        "  | A1 | A2 | B1 | B2 |\n",
+        "  |----|----|----|----|\n",
+        "  |  1 |  X | 10 |  P |\n",
+        "  |  1 |  X | 20 |  Q |\n",
+        "  |  2 |  Y | 10 |  P |\n",
+        "  |  2 |  Y | 20 |  Q |\n",
+        "\n",
+        "Cross-joins can be computationally expensive for large datasets, so use them judiciously.\n"
```

Comment:
Could you combine this cell and the next one into a single cell? I think they're discussing the same thing.